src/site/content/en/blog/notification-triggers/index.md

```diff
@@ -132,29 +126,29 @@ const createScheduledNotification = async (tag, title, timestamp) => {
 
 To cancel scheduled notifications, first request a list of all notifications that match a
 certain tag through `ServiceWorkerRegistration.getNotifications()`. Note that you need to pass the
-`includeTriggered` flag for scheduled notifications to be included in the list.
+`includeTriggered` flag for scheduled notifications to be included in the list:
 
-```js
+```js/4
 const cancelScheduledNotification = async (tag) => {
-  const registration = await navigator.serviceWorker.getRegistration();  
+  const registration = await navigator.serviceWorker.getRegistration();
   const notifications = await registration.getNotifications({
     tag: tag,
     includeTriggered: true,
-  }); 
+  });
   notifications.forEach((notification) => notification.close());
 };
 ```
 
 ### Debugging
 
-You can use the [Chrome DevTools Notifications pane][devtools] to debug notifications.
+You can use the [Chrome DevTools Notifications panel][devtools] to debug notifications.
 To start debugging, press **Start recording events** ![Start recording events](record.png) or
-<kbd>Control</kbd>+<kbd>E</kbd> (<kbd>Command</kbd>+<kbd>E</kbd> on Mac). Chrome DevTools
+`Control+E` (`Command+E` on Mac). Chrome DevTools
```

Comment:
@jpmedley the `kbd` tag is the correct formatting to use here

Reply:
```suggestion
kbd>Control</kbd>+<kbd>E</kbd> (<kbd>Command</kbd>+<kbd>E</kbd> on Mac). Chrome DevTools
```

Reply:
Hey, @kaycebasques, just a heads-up that the GDDSG [advises to use backticks in MD for keyboard keys](https://developers.google.com/style/ui-elements#keyboard). If you want to depart from that for web.dev, might be good to make a note in the handbook.